apps/studio/components/interfaces/Auth/EmailTemplates/EmailTemplates.tsx

```diff
@@ -43,32 +55,59 @@ export const EmailTemplates = () => {
               <EmailRateLimitsAlert />
             </div>
           ) : null}
-          <Card>
-            <Tabs_Shadcn_ defaultValue={TEMPLATES_SCHEMAS[0].title.trim().replace(/\s+/g, '-')}>
-              <TabsList_Shadcn_ className="pt-2 px-6 gap-5 mb-0 overflow-x-scroll no-scrollbar mb-4">
-                {TEMPLATES_SCHEMAS.map((template) => {
-                  return (
-                    <TabsTrigger_Shadcn_
-                      key={`${template.id}`}
-                      value={template.title.trim().replace(/\s+/g, '-')}
-                    >
-                      {template.title}
-                    </TabsTrigger_Shadcn_>
-                  )
-                })}
-              </TabsList_Shadcn_>
+          {isSecurityNotificationsEnabled ? (
+            <Card>
               {TEMPLATES_SCHEMAS.map((template) => {
-                const panelId = template.title.trim().replace(/\s+/g, '-')
+                const templateSlug = template.title.trim().replace(/\s+/g, '-').toLowerCase()
```

Comment:
Kind of annoying: our definitions for each template don’t have much information, and the IDs are a `LIKE_THIS`. So it requires a bit of futzing in client-side code, like this. See [AuthTemplatesValidation](https://github.com/supabase/supabase/blob/dnywh/chore/auth-template-subpages/apps/studio/components/interfaces/Auth/AuthTemplatesValidation.tsx#L0-L1)